nativeauthenticator/handlers.py

```diff
@@ -31,22 +31,37 @@ class SignUpHandler(LocalBase):
         html = self.render_template('signup.html')
         self.finish(html)
 
+    def get_result_message(self, message_type):
+        messages = {
+            'normal': 'Your information have been sent to the admin',
+            'error': (f'Something went wrong. Be sure your password has at '
+                      f'least {self.authenticator.minimum_password_length} '
+                      f'characters and is not too common.'),
```

Comment:
```
var = ('string part1'
       'string part2'
       'string part3')

# is the same as

var = 'string part1' + 'string part2' + 'string part3'
```
So in this case, you only need `f` in the second part. Maybe using `.format()` is simpler.